packages/template-typescript-minimal/package.json

```diff
@@ -38,8 +38,10 @@
     "npm": "^8.0.0 || ^9.0.0 || ^10.0.0"
   },
   "mobify": {
-    "app":{
-      "extensions": ["@salesforce/extension-sample"]
+    "app": {
+      "extensions": [
+        ["@salesforce/extension-sample", {"path": "/foo-page"}]
```

Comment:
By default, every extension has the config `enabled: true`. The extension developer do not need to handle it in their code, but we'll take care of it in our SDK. When we iterate through all of the extensions, we'll check whether they're enabled or not.